ext/rp/pico.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022, Andrey Kunitsyn
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+/**
+ * Pico SDK compatibility for tinyusb
+ */
+
+
+#include <stdint.h>
+#include <assert.h>
```

Comment:
```suggestion
#include <modm/architecture/interface/assert.h>
```
Otherwise Newlib's implementation get's pulled in and it's really expensive.

Reply:
Ah, I missed that you're overwriting the assert with the macro below of course, very good!